lib/options/dcu.robot

```diff
@@ -33,7 +34,7 @@ Set UEFI Option
     Should Contain    ${result.stdout}    Success
     SSHLibrary.Put File    dcu/coreboot.rom    coreboot.rom
     ${result}=    Execute Command In Terminal
-    ...    flashrom -p internal -w coreboot.rom --fmap -i SMMSTORE --noverify-all &> /dev/null
+    ...    flashrom -p internal -w coreboot.rom --fmap -i SMMSTORE    300s
```

Comment:
wont doping the `noverify-all` flag cause more tear in the chip? Or is there a reason to always flash whole die?

Reply:
Verifying will certainly wear the chip more, but the test will not continue if the flashing failed in an obvious way. If the wear is a real concern, then this change can be omitted at the cost of clearer logs in case of failed flashing.

Reply:
Verifying is reading so the added wear, if any, is miniscule. Even if it was writing the entire chip I don't think we need to concern ourselves with flash wear-out